qt/widgets/common/src/AlgorithmPropertiesWidget.cpp

```diff
@@ -312,10 +318,11 @@ void AlgorithmPropertiesWidget::replaceWSClicked(const QString &propName) {
       // Choose from candidates, only do this if there are candidates to select
       // from.
       if (candidateReplacementSources.size() > 0) {
-        CollectionOfPropertyWidget::iterator selectedIt = std::find_if(
-            candidateReplacementSources.begin(), candidateReplacementSources.end(), isCalledInputWorkspace);
+        CollectionOfPropertyWidget::iterator selectedIt =
+            std::find_if(candidateReplacementSources.begin(), candidateReplacementSources.end(),
```

Comment:
We can refactor this existing code to use const iterators e.g.

```suggestion
        const auto selectedIt =
            std::find_if(candidateReplacementSources.cbegin(), candidateReplacementSources.cend(),
```